src/styles/index.scss

```diff
@@ -1,3 +1,150 @@
-body {
+* {
+  box-sizing: border-box;
+}
+
+body,
+p,
```

Comment:
Since this is a small task, let's follow the coding guide and only use `html` and `body` for CSS reset. For other elements in this task we can use classes.